db/revision_cache_lru.go

```diff
@@ -152,7 +153,8 @@ func (rc *LRURevisionCache) UpdateDelta(ctx context.Context, docID, revID string
 	if value != nil {
 		outGoingBytes := value.updateDelta(toDelta)
 		if outGoingBytes != 0 {
-			rc.updateRevCacheMemoryUsage(outGoingBytes)
+			rc.currMemoryUsage.Add(outGoingBytes)
+			rc.cacheMemoryBytesStat.Add(outGoingBytes)
```

Comment:
I think we can just do this here, right?

```suggestion
			rc.incrRevCacheMemoryUsage(outGoingBytes)
```

Reply:
My predicament here is `outGoingBytes` could be incrementing bytes count or decrementing it depending on how the updating the delta can effect the overall memory footprint of the value.

Reply:
Ah gotcha - didn't realise outGoingBytes could be positive or negative.